api/features/import_export/views.py

```diff
@@ -0,0 +1,93 @@
+import json
+
+from django.db.models import QuerySet
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.generics import ListAPIView, get_object_or_404
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from environments.models import Environment
+
+from .models import FeatureExport
+from .permissions import (
+    CreateFeatureExportPermissions,
+    DownloadFeatureExportPermissions,
+    FeatureExportListPermissions,
+    FeatureImportPermissions,
+)
+from .serializers import (
+    CreateFeatureExportSerializer,
+    FeatureExportSerializer,
+    FeatureImportSerializer,
+    FeatureImportUploadSerializer,
+)
+
+
+@swagger_auto_schema(
+    method="POST",
+    request_body=CreateFeatureExportSerializer(),
+    responses={201: CreateFeatureExportSerializer()},
+)
+@api_view(["POST"])
+@permission_classes([CreateFeatureExportPermissions])
+def create_feature_export(request: Request) -> Response:
+    serializer = CreateFeatureExportSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+    serializer.save()
+
+    return Response(serializer.validated_data, status=201)
+
+
+@swagger_auto_schema(
+    method="POST",
+    request_body=FeatureImportUploadSerializer(),
+    responses={201: FeatureImportSerializer()},
+)
+@api_view(["POST"])
+@permission_classes([FeatureImportPermissions])
+def feature_import(request: Request, environment_id: int) -> Response:
+    upload_serializer = FeatureImportUploadSerializer(data=request.data)
+    upload_serializer.is_valid(raise_exception=True)
+    _feature_import = upload_serializer.save(environment_id=environment_id)
```

Comment:
I changed the name of this variable to stop my IDE complaining about it shadowing the name from the outer scope.